### PR TITLE
Add ENABLED flag in Notifcations and Job Alerts APIs

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
### Story Link
https://edlyio.atlassian.net/browse/COL-142

### Description
Add ENABLED flag in Notifications and Job Alerts APIs

### CODE REVIEW

File: openedx/features/colaraz_features/api/views.py [hunk 1]
```diff
@@ -90,9 +91,9 @@ def get(self, request, api_method, format=None):
             else:
                 LOGGER.error('Notifications API returned {} status'.format(resp.status_code))
         else:
-            LOGGER.error('Notifications API parameters are not complete or configured properly')
+            LOGGER.error('Notifications API is not enabled or configured properly')
```
Comment (danialmalik) [inline openedx/features/colaraz_features/api/views.py:94]:
```suggestion
            LOGGER.error('Notifications API is not enabled or is not configured properly')
```

Comment (HamzaIbnFarooq) [reply openedx/features/colaraz_features/api/views.py]: Updated.

File: openedx/features/colaraz_features/api/views.py [hunk 2]
```diff
@@ -68,10 +68,11 @@ def get(self, request, api_method, format=None):
...
         """
         elgg_id = request.user.colaraz_profile.elgg_id
-        api_details = getattr(settings, 'COLARAZ_NOTIFICATIONS')
-        method_key = self.API_METHODS.get(api_method, None)
+        api_details = getattr(settings, 'COLARAZ_NOTIFICATIONS', {})
+        is_enabled = api_details.get('ENABLE', False)
+        method_key = self.API_METHODS.get(api_method)
 
-        if api_details and elgg_id and method_key:
+        if api_details and is_enabled and elgg_id and method_key:
```
Comment (imhassantariq) [inline openedx/features/colaraz_features/api/views.py:75]: I think `is_enabled` only going to `True` if `api_details` have some value we can remove `api_details` from condition.

```suggestion
        if is_enabled and elgg_id and method_key:
```

Comment (HamzaIbnFarooq) [reply openedx/features/colaraz_features/api/views.py]: Updated.

**Pull request merged**